bridges/snowbridge/README.md

```diff
@@ -0,0 +1,120 @@
+# Snowbridge &middot; [![codecov](https://codecov.io/gh/Snowfork/snowbridge/branch/main/graph/badge.svg?token=9hvgSws4rN)](https://codecov.io/gh/Snowfork/snowbridge) ![GitHub](https://img.shields.io/github/license/Snowfork/snowbridge)
+
+Snowbridge is a trustless bridge between Polkadot and Ethereum. For documentation, visit https://docs.snowbridge.network.
+
+## Components
+
+### Parachain
+
+Polkadot parachain and our pallets. See [parachain/README.md](parachain/README.md).
+
+### Contracts
+
+Ethereum contracts and unit tests. See [contracts/README.md](contracts/README.md)
```

Comment:
Maybe worth changing relative paths to point inside`https://github.com/Snowfork/snowbridge`? Most of stuff is missing from the subtree

Reply:
Updated here: https://github.com/paritytech/polkadot-sdk/pull/2522/commits/bac0c6fc32f2f328ae925c8cfdf831ed020fd0bb